tests/telemetry/test_telemetry.py

```diff
@@ -458,6 +464,72 @@ def test_validate_entries(monkeypatch):
     assert res == (event_id, uid, action, client_time, elapsed_time)
 
 
+def test_get_latest_version(monkeypatch):
+    is_latest = telemetry.get_latest_version()
+    assert isinstance(is_latest, str)
+    version_index = [i for i, ltr in enumerate(is_latest) if ltr == '.']
+    assert len(version_index) == 2
```

Comment:
what are these two assert statements checking? they don't seem important, the latest one does `assert is_latest == ploomber.__version__`

Reply:
So it's to test that the pipy http call returned a valid version of type string and it has the right formatting either 1 or 2 .'s